MovingBlocks/Terasology
# refactor(translation): use `ResourceUrn` instead of `SimpleUrn`

### Contains
Prepare to extract I18n as Subsystem.
Part of #4304.
It required for extract AutoConfig. 

### How to test

1. Run main menu. See that text rendered normally
2. Run game. See that text rendered normally.
